packages/dropdown/src/DropdownItem/DropdownItem.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import { InputOption, InputOptionContent } from '@leafygreen-ui/input-option';
+import { keyMap } from '@leafygreen-ui/lib';
+import { InferredPolymorphic } from '@leafygreen-ui/polymorphic';
+
+import { useDropdownContext } from '../DropdownContext';
+import { useFocusableDropdownItem, useMergeRefs } from '../utils';
+
+import { DropdownItemProps } from './DropdownItem.types';
+
+export const DropdownItem = InferredPolymorphic<DropdownItemProps, 'div'>(
+  (
+    {
+      children,
+      disabled,
+      active,
+      description,
+      leftGlyph,
+      rightGlyph,
+      onClick,
+      ...rest
+    }: DropdownItemProps,
+    forwardRef,
+  ) => {
+    const { handleDropdownClose, renderedContext } = useDropdownContext();
+
+    const {
+      ref,
+      index,
+      onFocus,
+      onBlur,
+      tabIndex,
+      ['data-selected']: dataSelected,
```

Comment:
why are we using a data attribute if this is being passed into the input option as the `highlight` prop?

Reply:
I can update this